versioned_docs/version-4.6/reference/caching.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Caching
+---
+
+# Caching
+Harper provides integrated caching features for external data sources. This page documents all schema directives, configuration options, and resource APIs relevant to caching.
```

Comment:
Our caching system is not just for external data sources. In fact we'd rather our users use Harper as the data source. But we support integration with 3rd party data sources because that is an important pattern for customers. Also why is this page documenting Resource api methods? Those have their own reference page. If you're trying to demonstrate how they integrate with a caching example I think thats better set for the guide page or its own example.